lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -38,7 +70,7 @@
     <button class="action action-primary">
         <%=
             interpolate_text(
-                _.escape(gettext("{primaryButtonTitle} {span_start}a new team{span_end}")),
```

Comment:
The line below (73) is going to need translator comments-- it would be confusing to a translator.

Also, does it matter if whitespace is put between {span_start} and "a team" (and then the {span_end})? If so, this is going to be very brittle for translation.